pkg/executor/executortype/poolmgr/gp.go

```diff
@@ -632,17 +682,43 @@ func (gp *GenericPool) getFuncSvc(ctx context.Context, fn *fv1.Function) (*fscac
 		Address:           svcHost,
 		KubernetesObjects: kubeObjRefs,
 		Executor:          fv1.ExecutorTypePoolmgr,
+		CPULimit:          cpuLimit,
 		Ctime:             time.Now(),
 		Atime:             time.Now(),
 	}
 
+	gp.podFSVCMap.Store(pod.ObjectMeta.Name, []interface{}{crd.CacheKey(fsvc.Function), fsvc.Address})
 	gp.fsCache.AddFunc(*fsvc)
 
 	gp.fsCache.IncreaseColdStarts(fn.ObjectMeta.Name, string(fn.ObjectMeta.UID))
 
 	return fsvc, nil
 }
 
+// get85Percent returns 85 percent of the quantity i.e multiple it by 17/20
+func (gp *GenericPool) get85Percent(cpuUsage resource.Quantity) (resource.Quantity, error) {
```

Comment:
@RealHarshThakur the method you suggested in this I would have to convert and parse suffix like milli, nano etc which would be tedious. 
@vishal-biyani I think this will be useful, this would make code cleaner.